pvlib/irradiance.py

```diff
@@ -3766,3 +3766,69 @@ def louche(ghi, solar_zenith, datetime_or_doy, max_zenith=90):
         data = pd.DataFrame(data, index=datetime_or_doy)
 
     return data
+
+
+def spitters_relationship(solar_zenith, global_diffuse_fraction):
+    r"""
+    Derive the diffuse fraction of photosynthetically active radiation (PAR)
+    respect to the global radiation diffuse fraction.
+
+    The relationship is based on the work of Spitters et al. (1986) [1]_.
+
+    Parameters
+    ----------
+    solar_zenith : numeric
+        Solar zenith angle. Degrees.
+
+    global_diffuse_fraction : numeric
+        Fraction of the global radiation that is diffuse. Unitless [0, 1].
+
+    Returns
+    -------
+    par_diffuse_fraction : numeric
+        Photosynthetically Active Radiation diffuse fraction. Unitless [0, 1].
+
+    Notes
+    -----
+    The relationship is given by equations (9) & (10) in [1]_ and (1) in [2]_:
+
+    .. math::
+
+        k_{diffuse\_PAR}^{model} = \frac{PAR_{diffuse}}{PAR_{total}} =
+        \frac{\left[1 + 0.3 \left(1 - \left(k_d^{model}\right) ^2\right)\right]
+        k_d^{model}}
+        {1 + \left(1 - \left(k_d^{model}\right)^2\right) \cos ^2 (90 - \beta)
+        \cos ^3 \beta}
+
+    where :math:`k_d^{model}` is the diffuse fraction of the global radiation,
+    provided by some model.
```

Comment:
```suggestion
    provided by some model, and :math:`\beta` is the solar zenith angle (degrees).
```

Reply:
Just so my commit doesn't surprise you, it is elevation:

> In Eq.1, the superscript model refers to the decomposition model applied to obtain the diffuse fraction of global radiation, 𝑘𝑑, and 𝛽 is the solar elevation angle [º]. Several models are analyzed and explained in the following section.